mods/action.js

```diff
@@ -155,6 +155,9 @@ ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
     body.action = defBody.action;
     body.format = body.format || defBody.format || 'json';
     body.formatversion = body.formatversion || defBody.formatversion || 1;
+    if (defBody.rawcontinue !== undefined) {
```

Comment:
We also might want to check (temporarily) for `req.body.continue`: if the caller provided that one, then we do not need to append `rawcontinue`